templates/malaria/edit_post.html

```diff
@@ -1,6 +1,4 @@
-{% include "infohub/index.html" %}
-
-{% block content %}
+{% include "infohub/index.html" %} {% block content %}
```

Comment:
Why this change?

Reply:
Still not a code change, It's just formatting which should not break anything